README.md

```diff
@@ -1,2 +1,133 @@
-# carbonite-migrate
-Migrate devices using Carbonite Migrate.
+carbonite-migrate
```

Comment:
We might not want to host the images on imgur.com since this is for commercial use.  It might be better to have these images hosted on our S3 instead.

Reply:
I've packaged up these all the images and will send them to you.